CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+- Added sliderRoot css handle
```

Comment:
Write the CHANGELOG according with the [spec](https://keepachangelog.com/en/1.0.0/)